pkg/components/components.go

```diff
@@ -524,15 +524,20 @@ func GetClusterPermissions() []rbacv1.PolicyRule {
 		},
 		roleWithAllPermissions("console.openshift.io", stringListToSlice("consoleclidownloads", "consolequickstarts")),
 		{
-			APIGroups: stringListToSlice("config.openshift.io"),
+			APIGroups: stringListToSlice(configOpenshiftIO),
 			Resources: stringListToSlice("clusterversions", "infrastructures", "ingresses"),
 			Verbs:     stringListToSlice("get", "list"),
 		},
 		{
-			APIGroups: stringListToSlice("config.openshift.io"),
+			APIGroups: stringListToSlice(configOpenshiftIO),
 			Resources: stringListToSlice("apiservers"),
 			Verbs:     stringListToSlice("get", "list", "watch"),
 		},
+		{
+			APIGroups: stringListToSlice(configOpenshiftIO),
+			Resources: stringListToSlice("dnses"),
+			Verbs:     stringListToSlice("get"),
```

Comment:
I hope it's enough. If it won't work, please consider to add also `list` (or even better, add it to the list in line 528).

Reply:
it worked with just get if thats what you mean
thanks !